internal/hooks/auth_hooks.go

```diff
@@ -141,9 +146,8 @@ type CustomAccessTokenOutput struct {
 }
 
 type SendSMSInput struct {
-	UserID uuid.UUID `json:"user_id"`
-	Phone  string    `json:"phone"`
-	OTP    string    `json:"otp"`
+	User      *models.User `json:"user"`
+	PhoneData PhoneData    `json:"phone_data"`
```

Comment:
```suggestion
	PhoneData PhoneData    `json:"phone_data,omitempty"`
```

Reply:
Also why is it called `phone_data`? Shouldn't it be just `sms`?

Reply:
Sure we can call it SMS